apischema/graphql/resolvers.py

```diff
@@ -269,7 +269,9 @@ def resolve(__self, __info, **kwargs):
         errors: Dict[str, ValidationError] = {}
         for alias, param_name, deserializer, opt_param, required in parameters:
             if alias in kwargs:
-                if not opt_param and kwargs[param_name] is None:
+                # If this parameter was required but not supplied we can ignore
```

Comment:
This comment is not exact (see my [comment](https://github.com/wyfo/apischema/pull/130#issuecomment-845497392) in the PR)
The parameter is not required but its default is not serializable, so it ends up with an optional parameter, with `null` as an authorized value. In that case, it should be ignored to ensure using the true default value.

Reply:
By the way, because the parameter is in `kwargs`, that means it has been explicitely supplied by the *GraphQL* query.

Reply:
I've redone the comment, is that closer to what you mean? Re: `kwargs`, do you mean as opposed to just having a default value?

Reply:
The comment is perfect.
About `kwargs`, your previous comment was not exact, because `kwargs[alias]` being `None` means that the argument **was supplied* in the query; if it was not supplied, `kwargs` would not contains `alias`.